user-guides/end-to-end-encryption/README.md

```diff
@@ -0,0 +1,65 @@
+# End to End Encryption
+
+> Note: This feature is currently in alpha.  Its also not yet supported on mobile
+
+## Getting started
+
+Once your servers admin has enabled End to End Encryption as an option you will see a banner across the top of your interface like this:
+
+![ ](e2e-banner.png)
+
+If you click this banner you will be presented with a dialog containing a password. Rocket.Chat server does not know this password. So you will need to store it in your password manager along with your login password.
+
+![ ](e2e-alert.png)
+
+You will need this password on any other device you wish to participate in end to end conversations.
+
+Once you have this password saved somewhere it is safe to dismiss the alert.
+
+## Changing Encryption Password
+
+If you would like to change your encryption password you can do so by going to: My Account->Encryption
+
+![ ](e2e-changepassword.png)
+
+Once you make this change you will need to update the password on any other devices being used.
```

Comment:
This part is not true, the user will need the new password only when importing the credentials again, usually in a new client

Reply:
ah.. good to know!